roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -194,6 +194,14 @@
 
 # storageclasses are used by default but if static then disable
 # storageclasses with the storageClassName set to "" in pvc.j2
+- name: Check to see if PVC already exists
+  oc_obj:
+    state: list
+    kind: pvc
+    name: "{{ openshift_logging_elasticsearch_pvc_name }}"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+  register: logging_elasticsearch_pvc
+
```

Comment:
LGTM however IMHO it would look even better if the tasks went in a block instead of duplicating the conditional.

```
   - when:
      - logging_elasticsearch_pvc.results.stderr is defined
      - openshift_logging_elasticsearch_storage_type == "pvc"
     block:
       [...]
```
That way you don't have to look far to see what `logging_elasticsearch_pvc.results.stderr` is all about. I wish there were a more readable way to indicate in the conditional what it's for but I guess there's not much to be done about that.